test/integration/route-href.spec.ts

```diff
@@ -1,11 +1,13 @@
-import {TemplatingRouteLoader} from 'src/route-loader';
-import {bootstrap} from 'aurelia-bootstrapper';
-import {RouteLoader, AppRouter, Router} from 'aurelia-router';
-import {StageComponent} from 'aurelia-testing';
+import { TemplatingRouteLoader } from '../../src/route-loader';
+import { bootstrap } from 'aurelia-bootstrapper';
+// import { bootstrap } from './bootstrap';
```

Comment:
what happened here

Reply:
lol. I was having a bit of issue converting the test to webpack, not sure what happened. It's all working now